hbook/ntup/ntup_example_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup
+
+import (
+	"fmt"
+	"log"
+	"go-hep.org/x/hep/hbook"
```

Comment:
please put an empty line before the `hbook` import to separate standard library imports from 3rd-party imports.
you'll probably also have to import `"go-hep.org/x/hep/hbook/ntup/ntcsv"` to be able to use it inside the example code.